### ISSUE TITLE
Enhancement: Displaying 'No items' on the 'Starting Items' screen.

### ISSUE THREAD

**Issue by chrismaliszewski** [NPBruce/valkyrie#986]
### Description of Problem or Question
Today when starting a scenario (Delve Too Deep) I didn't receive any Starting Items. When I saw the screen, I was wondering if it's an error (i.e. items were not displayed) or everything is as it should be.

I don't think removing the step altogether when players don't receive any items would be a good idea. Therefore, I would say a statement like 'You don't receive any items' should be displayed for clarity that there is no error or something like that. 

### Valkyrie Version
2.15, @BenRQ 's 2.2.20181209

**Comment by scrubbless** [comment]
The step is there to give a place for investors to get their starting items regardless if the scenario gives them or not.


A 'dont have any items' statement would be good though.

**Comment by chrismaliszewski** [comment]
Yeah, I know what is the purpose of this step. If you add the statement, I'd be great!

**Comment by BenRQ** [comment]
I'll need the translation in all languages to do this.

I'm afraid "don't have any items" may have a strange translation though if using automatic translation.
"You don't receive any items" is probably better to translate through Google

**Comment by chrismaliszewski** [comment]
Agree. I just wrote a minor suggestion for the text. Not something binding :). Do whatever you find best but I think our proposal is ok.

**Comment by chrismaliszewski** [comment]
PL:
Nie otrzymasz żadnych przedmiotów startowych
